ledger-tool/src/main.rs

```diff
@@ -787,17 +787,6 @@ fn main() {
             .arg(&halt_at_slot_arg)
             .arg(&hard_forks_arg)
             .arg(&max_genesis_archive_unpacked_size_arg)
-        ).subcommand(
-            SubCommand::with_name("prune")
```

Comment:
Turns out this is a ledger-tool command we very likely don't need.  Removing it makes it easy to make the `to_slot`  argument of `purge_slots()` non-optional